src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -639,6 +635,14 @@ private VariantContext filterSiteByAlleleSpecificVQSLOD(VariantContext mergedVC,
         return filteredVC;
     }
 
+    boolean isFailingSite(final Stream<Double> vqScores, final Double vqScoreThreshold) {
+        Optional<Double> maxVal = vqScores
+                .filter(d -> !(d.isNaN()||d.isInfinite()))
+                .max(Double::compareTo);
+        return maxVal.isPresent() && maxVal.get() < vqScoreThreshold;
+    }
```

Comment:
not necessary for this pr, but I think additional documentation around this is necessary.
also lol do we really get vqscores of "infinite"?

Reply:
Yeah, that's a good point. Not sure if we'd ever see infinite, but am hesitant to pull that check out.